addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -492,19 +492,52 @@ const Wysiwyg = Widget.extend({
      *
      * @param {object} [params]
```

Comment:
You removed params but not the docstring mentioning them.

Reply:
:+1: